src/hba.c

```diff
@@ -627,15 +639,17 @@ static bool parse_ident_line(struct Ident *ident, struct TokParser *tp, int line
 	if (find_ident_map(ident, map_name_copy, &ident_map)) {
 		list_append(&ident_map->mappings, &mapping->node);
 		free(map_name_copy);
+		/* map_name_copy = NULL; */
```

Comment:
It looks ok that this can be uncommented.  Although setting to `NULL ` after `free ` is not the convention throughout the code base, this case is somewhat special as the `free `is called in mid function.